pkg/proxy/winkernel/hcnutils.go

```diff
@@ -50,6 +50,7 @@ type HcnService interface {
 	// Policy functions
 	DeleteAllHnsLoadBalancerPolicy()
 	RemoteSubnetSupported() error
+	IsNotImplemented(err error) bool
```

Comment:
Is 'IsNotImplemented' an actual Windows API function? If something is not implemented, isnt this a function level error? Example, kube-proxy called CreateLoadBalancer and the specific kernel did not implement it would return error stating this? This looks like host container network does in fact implement this?